recon_test_pack/ROOT_STIR_consistency/run_pretest_script.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+for I in {1..12}
+do
+	generate_image generate_image${I}.par
+	cd Gate_macros
+	# Create main GATE macro files from template
+	sed -e s/SOURCENAME/test${I}/ main_D690_template.mac > main_D690_test${I}.mac
```

Comment:
`sed` issues may occur on MacOS. I personally have to use `gsed` for this command to work. This is documented [elsewhere](https://unix.stackexchange.com/a/131940) but may lead to issues for us in this case.

Reply:
it's true, but the pretest script is only run to create the files, so presumably only needs running once in a while. Let's live with it (maybe just add a TODO comment in the file)

Reply:
Agreed. The hroot and root files can be downloaded. `run_pretest_script.sh` does not need to be run for each test.